<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_goodsEdit</class>
 <widget class="QDialog" name="Admin_goodsEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_goodsID">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>正在修改货物：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_goodsName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_goodsName">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货品名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_storageNum">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>库存数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_storageNum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_place">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>存放地点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_place">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_producer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>制造商：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_producer">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
